cnxarchive/views.py

```diff
@@ -603,7 +603,7 @@ def in_book_search(request):
             for uuid, version, title, headline, rank in res:
                 results['results']['items'].append({
                     'rank': '{}'.format(rank),
-                    'id': '{}@{}'.format(uuid, version),
+                    'id': '{}@{}'.format(uuid, version.split('.')[1]),
```

Comment:
Yeah, I spotted this and suggested @mwharrison fix it a different way. Push should be here in a minute.